CGPs/cgp-0128/mainnet.json

```diff
@@ -0,0 +1,25 @@
+[
+    {
```

Comment:
This is missing the transfer from the reserve, right?

Reply:
Not really since we are specifying range of stCelo<->Celo, but the pool already exists and Uniswap doesn't allow us to create LP with 1:1